packages/gatsby-source-contentful/src/__tests__/__snapshots__/gatsby-node.js.snap

```diff
@@ -1,15 +1,15 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`gatsby-node stores rich text as raw with references attached 1`] = `
-Array [
+[
```

Comment:
These snapshot tests were failing for me. I'm not sure if it's a mismatch in jest version. Let's see if it fails in CI....